static/js/lib/courseApi.js

```diff
@@ -21,11 +21,7 @@ export const isLinkableCourseRun = (
     return true
   }
   const now = dtNow || moment()
-  return (
-    notNil(run.start_date) &&
-    moment(run.start_date).isBefore(now) &&
-    (isNil(run.end_date) || moment(run.end_date).isAfter(now))
-  )
+  return notNil(run.start_date) && moment(run.start_date).isBefore(now)
```

Comment:
@arslanashraf7 the [story#346](https://github.com/mitodl/mitxonline/issues/346) is pointing to say `Past courses are no longer available on dashboard` but the change here is particularly relates to either making course over dashboard Linkable or Not. 
I believe, We added this condition `isLinkableCourseRun ` in past to fix this story [Don't link to courses that aren't open yet](https://github.com/mitodl/mitxonline/pull/139).  

@umarmughal824 as you are reviewing this so please take care of that.

Reply:
After going through the story details, It seems title of it is bit confusing `Past courses are no longer available on dashboard` but the question is standing same here about the past fix for [Don't link to courses that aren't open yet.](https://github.com/mitodl/mitxonline/pull/139)

Reply:
@briangrossman can you confirm either we should not take care of `end_date` [previously](https://github.com/mitodl/mitxonline/pull/139) too as per discussion with Gavin, he pointing out to take care of End date as well earlier.